fighthealthinsurance/views.py

```
@@ -2192,6 +2179,107 @@ def form_valid(self, form):
+                filename=safe_filename,
+                hashed_email=hashed_email,
+                session_key=session_key,
+            )
+
+            logger.info(f"Created PolicyDocument {policy_doc.id} for {safe_filename}")
+
+            # Store the document ID in session for chat to pick up
+            self.request.session["policy_document_id"] = str(policy_doc.id)
+            self.request.session["policy_document_question"] = user_question
+
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)